crates/node-primitives/src/lib.rs

```diff
@@ -58,4 +58,6 @@ pub enum CallError {
     ApplicationNotInstalled { application_id: ApplicationId },
     #[error("internal error")]
     InternalError,
+    #[error("Alias error")]
+    AliasResolutionFailed,
```

Comment:
it would be useful to mention which alias we had an issue with

```suggestion
    #[error("error resolving identity alias '{alias}'")]
    AliasResolutionFailed {
        alias: Alias<PublicKey>,
    },
```